doc/radosgw/qat-accel.rst

```diff
@@ -92,3 +152,4 @@ Edit the Ceph configuration file to enable QAT support for compression::
 .. _QATzip: https://github.com/intel/QATzip
 .. _OpenSSL support for RGW encryption: https://github.com/ceph/ceph/pull/15168
 .. _QAT Engine: https://github.com/intel/QAT_Engine
+.. _build Ceph: https://docs.ceph.com/en/latest/install/build-ceph/#id1
```

Comment:
i think a more future-proof way to reference a section in this project is 

1. add a anchor at the place being referenced, something like
   ```rst
   .. _build-ceph:
   Build Ceph
   ==========
   ```
2.  and reference it like
   ```rst
   :ref:`build-ceph`
   ```

Reply:
Thanks for reminding.